package.json

```diff
@@ -32,55 +39,40 @@
 		"css"
 	],
 	"author": "RoelN",
-	"license": "ISC",
+	"license": "Apache-2.0",
```

Comment:
Generally speaking: A change in license would have to be confirmed by each contributor. If you don't get the buy-in their contributions have to be removed.

Some licenses allow for re-licensing.

That being said, the LICENSE file was added four years ago and should trump the entry in the `package.json`. You might want to check how it is presented on npm website (and other places that harvest npm for dynamic content).

Reply:
I don't know where "ISC" came from as it was Apache-2.0 from the beginning. So this isn't so much a change of license, as just mentioning the _correct_ license.